node/service/Cargo.toml

```diff
@@ -144,6 +144,7 @@ default = [ "kusama-native", "westend-native", "rococo-native" ]
 kusama-native = [ "polkadot-service/kusama-native", "polkadot-cli/kusama-native" ]
 westend-native = [ "polkadot-service/westend-native", "polkadot-cli/westend-native" ]
 rococo-native = [ "polkadot-service/rococo-native", "polkadot-cli/rococo-native" ]
+evm-tracing = [ "moonbeam-runtime/evm-tracing", "moonriver-runtime/evm-tracing", "moonbase-runtime/evm-tracing" ]
```

Comment:
It would be better to include the `moonbeam-rpc-trace` dependency only when this feature is enabled.

Reply:
Just pushed, a bit tricky. Let me know if that's what you had in mind.

Reply:
That's what I was thinking. But you can go further and have only one tracing module, present when there is the feature, it will be cleaner. I will guide you through some tips :)

Reply:
Sounds good, maybe we can refactor in a followup PR?

Reply:
It's not very clean to have 2 tracing modules, there is not much to do to have only one, if you don't have time I can add a commit :)

Reply:
Please do!

Reply:
Done